sdk/communication/azure-communication-rooms/src/main/java/com/azure/communication/rooms/implementation/converters/RoomParticipantConverter.java

```diff
@@ -17,15 +17,13 @@ public static RoomParticipant convert(com.azure.communication.rooms.implementati
             return null;
         }
 
-        RoomParticipant roomParticipant = new RoomParticipant()
-            .setCommunicationIdentifier(CommunicationIdentifierConverter.convert(obj.getCommunicationIdentifier()))
-            .setRole(RoleTypeConverter.convert(obj.getRole()));
+        RoomParticipant roomParticipant = new RoomParticipant(CommunicationIdentifierConverter.convert(obj.getRawId()), ParticipantRoleConverter.convert(obj.getRole()));
```

Comment:
We should be using the `converter` from common and removing our custom implementation of CommunicationIdentifierConverter.
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/communication/azure-communication-common/src/main/java/com/azure/communication/common/CommunicationIdentifier.java#L51